doc/syntax.org

```diff
@@ -0,0 +1,43 @@
+#+TITLE:   Markup Syntax Guide for Pancacke(d)
+#+AUTHOR:  Scott Wang
+#+STARTUP: align indent
+#+OPTIONS: H:5
+
+
+* Overview
```

Comment:
Should be in *Pancake-Flavored Org Mode*, the source is at https://github.com/weflex/pancake/blob/master/lib/pancake.js#L152